pallets/ds-accounts/src/default_weight.rs

```diff
@@ -1,23 +1,16 @@
 use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
 
 impl crate::WeightInfo for () {
```

Comment:
Why is this trait implemented for `()` type? Is this a good practice in Substrate to abuse irrelevant types? Can it be a dedicated newtype instead?

Reply:
default implementation for () is common practice for substrate. So, we can use 
`type SomeType = ();`

Reply:
Ok.